### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by arturo-seijas**
Add readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+This repository includes several parametrized CI workflows to be used for both kubernetes and machine charms.
+
+## Usage
+
+To use the workflows in your repository, just reuse them including the `secrets: inherit` key. Examples of usage for every workflow can be found at the (indico operator)[https://github.com/canonical/indico-operator/blob/main/.github/workflows].
+
+The following workflows are available:
+
+* test: executes the default tox targets defined in the `tox.ini`file and generates a plain text report. The following parameters are available for this workflow:
```
Comment (mthaddon) [inline README.md:13]: Looks like you're missing a whitespace after `tox.ini`.

**Review by mthaddon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (arturo-seijas) [reply README.md]: Indeed. Thx

**Review by weiiwang01 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+| Name | Type | Default | Description
+| runs-on| string | ubuntu-20.04 | Image runner for the test execution |
+
+* comment: Posts the content of the artifact specified as a comment in a PR. It needs to be triggered from a PR triggered workflow.
+
+* integration_test: Builds the existing Dockerfiles, if any, and executes the integration test target defined in the `tox.ini` file. The following parameters are available for this workflow:
```
Comment (gtrkiller) [inline README.md:20]: LGTM, but do you think it would be advisable to keep an eye on readme's line length?

**Review by mthaddon — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (arturo-seijas) [reply README.md:20]: It seems to fit the diff window :)

**Pull request merged**